SPECS/util-linux/util-linux.spec

```diff
@@ -83,6 +83,8 @@ autoreconf -fi
 make %{?_smp_mflags}
 
 %install
+chmod ug-s bash-completion/mount
```

Comment:
Could you add a comment explaining these? (Generally we should only be using `chmod` on files in `%{buildroot}`